### ISSUE TITLE
[Submit add-on]: Sound Splitter, stable translation update

### ISSUE THREAD

**Issue by XLTechie** [nvaccess/addon-datastore#3455]
### Download URL

https://github.com/opensourcesys/soundSplitter/releases/download/v24.0.1/soundSplitter-24.0.1.nvda-addon

### Source URL

https://github.com/opensourcesys/soundSplitter

### Publisher

Luke Davis

### Channel

stable

### License Name

GPL v2

### License URL

https://www.gnu.org/licenses/gpl-2.0.html

**Comment by XLTechie** [comment]
@nvdaes Any thoughts on this one?

```
Upload results
0s
1
Run actions/upload-artifact@v4
23
With the provided path, there will be 1 file uploaded
24
Artifact name is valid!
25
Root directory input is valid!
26
Error: Failed to CreateArtifact: Received non-retryable error: Failed request: (409) Conflict: an artifact with this name already exists on the workflow run
```

**Comment by nvdaes** [comment]
Hi @XLTechie , I've fixed this possible error in the not merged PR #3397
